src/web-worker/amp-worker.js

```diff
@@ -103,6 +106,17 @@ class AmpWorker {
       })
       .then((res) => res.text())
       .then((text) => {
+        // Replace sourceMappingUrl with the absolute URL
+        const base = calculateEntryPointScriptBase(
+          loc,
+          useLocal,
+          useRtvVersion
+        );
+        text = text.replace(
+          '//# sourceMappingURL=',
+          `//# sourceMappingURL=${base}/`
+        );
```

Comment:
We actually need to replace everything until the next newline:

```suggestion
        text = text.replace(
          /^\/\/# sourceMappingURL=.*/,
          `//# sourceMappingURL=${sourceMappingUrl}`
        );
```